.circleci/config.yml

```diff
@@ -198,20 +198,31 @@ jobs:
 
               # make sure we have a lowercase repo
               user_name="bids"
-              repo_name=$(echo "${CIRCLE_PROJECT_REPONAME}_fs<< parameters.version >>" | tr '[:upper:]' '[:lower:]')
+              repo_name=$(echo "${CIRCLE_PROJECT_REPONAME}" | tr '[:upper:]' '[:lower:]')
+              FSMAJORVERSION=<< parameters.version >>
```

Comment:
this is getting hard to read

should we go for SCREAMING_SNAKE_CASE?

FSVERSION -> FS_VERSION
FSMAJORVERSION -> FS_MAJOR_VERSION

Reply:
Sure, fine by me.